utils/config-schema.json

```diff
@@ -122,6 +122,12 @@
           "properties": {
             "contract": {
               "$ref": "#/definitions/ethaddress"
+            },
+            "lendingPool": {
+              "$ref": "#/definitions/ethaddress"
+            },
+            "aaveToken": {
```

Comment:
There are several aTokens so I think this should be an array `aTokens` of addresses.

Reply:
Okay, only did that because Tom was saying we only support aETH atm. Is that not the case?